projects/batfish/src/org/batfish/main/Batfish.java

```diff
@@ -3578,6 +3578,20 @@ public void pushDeltaEnvironment() {
       return configurationData;
    }
 
+   static List<Path> listAllFiles(Path configsPath) {
+      List<Path> configFilePaths;
+      try {
+         configFilePaths = Files.walk(configsPath)
+                 .filter(path -> !path.getFileName().toString().startsWith(".")
+                         && Files.isRegularFile(path))
+                 .sorted()
+                 .collect(Collectors.toList());
+      } catch (IOException e) {
+         throw new BatfishException("Failed to walk path: " + configsPath.toString(), e);
```

Comment:
I believe the `.toString()` is redundant as `"" + configsPath` will convert `configsPath` to String automatically. Drop it?